homeassistant/components/androidtv_remote/__init__.py

```diff
@@ -57,7 +57,9 @@ def is_available_updated(is_available: bool) -> None:
 
     def reauth_needed() -> None:
         """Start a reauth flow if Android TV is hard reset while reconnecting."""
-        entry.async_start_reauth(hass)
+        hass.async_create_background_task(
+            entry.async_init_reauth(hass), f"reauth flow {DOMAIN} {entry.entry_id}"
+        )
```

Comment:
The comment about the race is missing here

Reply:
I don't think it can race here.